ui/app/components/tx-list-item.js

```diff
@@ -28,6 +29,13 @@ TxListItem.prototype.render = function () {
     className,
   } = this.props
 
+  const txStatusClassNames = hyperClassnames('span.tx-list-status', {
+    '.tx-list-status--rejected': transactionStatus === 'rejected'
+  })
+  const txAmountClassNames = hyperClassnames('span.tx-list-value', {
```

Comment:
Good thought!

But this seems extraneous -- we can simply use classnames and pass it in regularly as props like this:

```js
h('div', { className: classnames(...)}, [...]);
```

Reply:
Change applied in ef6168bc